crates/breez-sdk/core/src/sdk.rs

```diff
@@ -1286,6 +1362,73 @@ impl BreezSdk {
         Ok(SendPaymentResponse { payment })
     }
 
+    async fn send_spark_token_address(
+        &self,
+        token_identifier: String,
+        amount: u128,
+        receiver_address: SparkAddress,
+    ) -> Result<Payment, SdkError> {
+        let token_transaction = self
+            .spark_wallet
+            .transfer_tokens(vec![TransferTokenOutput {
+                token_id: token_identifier,
+                amount,
+                receiver_address: receiver_address.clone(),
+                spark_invoice: None,
+            }])
+            .await?;
+
+        self.map_and_persist_token_transaction(&token_transaction)
+            .await
+    }
+
+    async fn send_spark_invoice(
+        &self,
+        invoice: &str,
+        _token_identifier: Option<String>,
```

Comment:
Can be removed?

Reply:
Yes! That was a leftover. Removed now. Also changed the implementation on the breez sdk level to force the inclusion of a token id in token invoices in order to consider them valid. That was the case in the spark wallet already.